### PR TITLE
Profile list response 

### PR DESCRIPTION
**Pull request by AnkurGel**
* Exposes `ProfileList` in `mdm.Response`

### CODE REVIEW

File: response.go [hunk 1]
```diff
@@ -226,3 +227,24 @@ type ErrorChainItem struct {
 }
 
 type ErrorChain []ErrorChainItem
+type PayloadContentItem struct {
+	PayloadDescription       string `json:"payload_description"`
+	PayloadDisplayName       string `json:"payload_display_name"`
+	PayloadIdentifier        string `json:"payload_identifier"`
+	PayloadOrganization      string `json:payload_organization`
```
Comment (groob) [inline response.go]: These json keys need quotes around the fields. Ex `json:"payload_organization"`

File: response.go [hunk 2]
```diff
@@ -226,3 +227,24 @@ type ErrorChainItem struct {
...
+}
+
+type ProfileListItem struct {
+	HasRemovalPasscode       bool                 `json:"has_removal_passcode"`
+	IsEncrypted              bool                 `json:"is_encrypted"`
+	PayloadContent           []PayloadContentItem `json:"payload_content,omitempty" plist:",omitempty"`
+	PayloadDescription       string               `json:"payload_description"`
+	PayloadDisplayName       string               `json:"payload_display_name"`
+	PayloadIdentifier        string               `json:"payload_identifier"`
+	PayloadOrganization      string               `json:payload_organization`
```
Comment (groob) [inline response.go]: JSON fields need quotes.

File: response_test.go [hunk 3]
```diff
@@ -119,3 +119,24 @@ func TestInstalledApplicationListResponse(t *testing.T) {
 
 	fmt.Printf("%v\n", response)
 }
+
+func TestProfileListResponse(t *testing.T) {
```
Comment (groob) [inline response_test.go:123]: Nice on tests!

File: response_test.go [hunk 4]
```diff
@@ -119,3 +119,24 @@ func TestInstalledApplicationListResponse(t *testing.T) {
...
+	if err != nil {
+		t.Fatal(err)
+	}
+	response := &Response{}
+	if err := plist.Unmarshal(profileListResponseBody, response); err != nil {
+		t.Fatal(err)
+	}
+
+	if response.Status != "Acknowledged" {
+		t.Fatal("Response status was not `Acknowledged`.")
```
Comment (groob) [inline response_test.go]: You probably want to change the t.Fatal to `t.Errorf`. The `Fatal` should only be used if the rest of the test cannot run, like if Unmarshal failed.

**Review by groob — COMMENTED** [review]
A few nits otherwise looks good. Once they're fixed, I'll merge them.

**Comment by AnkurGel** [comment]
Fixed

**Comment by groob** [comment]
That was fast 👍

**Pull request merged**